src/config/locales.md

```diff
@@ -14,7 +16,8 @@ To enable a certain locale, un-comment or add the relevant lines in
 
 ## Setting the system locale
```

Comment:
this heading is inaccurate, it's for setting the system *language*

Reply:
Fixed.